### ISSUE TITLE
JetBrains: Improve agent support for certs

### ISSUE THREAD

**Issue by chwarwick** [sourcegraph/sourcegraph#56984]
When a users has added additional certs to JetBrains due to proxies for outgoing requests the initial account setup api calls work however they are not known to the agent and there for any api requests from the agent will fail.  This can be difficult to debug.

The setting `cody.accept-non-trusted-certificates-automatically` allows the agent to accept non trusted certs however it is not exposed in the settings UI and requires setting via the `.sourcegraph-jetbrains.properties` file.  

There should be a setting in the UI that allows the users to either allow non trusted certificates for the agent or specify certificates that are allowed when starting the agent process.

**Comment by taylorsperry** [comment]
Reported by several customers who have upgraded to the latest release. I'm marking as a P0.

**Comment by chwarwick** [comment]
Additional details:

- During account setup I did not see any visual notification of any failure creating the account it ended up in the account list as if it was created successfully.  There was an error recorded in the idea.log
- ✅  In Tools > Server Certificates item "Accept non-trusted certificates automatically"  unblocked the the account creation step debug logs showed that the agent then started up and started reporting untrusted cert errors 
- creating a `~/.sourcegraph-jetbrains.properties` file with the `cody.accept-non-trusted-certificates-automatically=true` and restarting IDE did not work, the same cert errors were recorded.
- I tested locally and it appears a regression occurred during the conversion to Kotlin where that property is no longer read correctly.

Similar certificate issues can be simulated by running `sg start` and attempting to connect the extension to the local https site.
